invenio_upgrader/upgrades/invenio_upgrader_2015_09_23_legacy_removal.py

```diff
@@ -1,131 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I thought we were starting from scratch, meaning no upgrade recipes. 
I will added then.